scenarioo-server/src/main/java/org/scenarioo/dao/diffViewer/GraphicsMagickConfiguration.java

```diff
@@ -24,10 +28,13 @@ private static boolean isGraphicsMagickAvailable() {
 		try {
 			gmConsole.run(gmOperation.version());
 		} catch (IOException e) {
+			LOGGER.warn(e, e);
```

Comment:
@danielsutter please provide a meaningful warning message with some context information instead of just logging (e,e) - not really helpfull to log the exception twice. Something like "DiffViewer: GraphicMagick not available, failed to access. No screenshot comparison will be available in DiffViewer."